### PR TITLE
Create game page layout

### PR DESCRIPTION
**Pull request by shandyk18**
Screenshot 2020-07-28 at 12 41 52 PM - Display 2

### CODE REVIEW

File: ubg/src/reviews/AllReviews.js [hunk 1]
```diff
@@ -17,6 +17,7 @@ let initialize = false;
  * @return {ReactElement} Box containing review section
  */
 function AllReviews(props) {
+  console.log(props.gameId);
```
Comment (danieltre23) [inline ubg/src/reviews/AllReviews.js:20]: Do we need this?

Comment (shandyk18) [reply ubg/src/reviews/AllReviews.js:20]: Nope. Deleting it.

File: ubg/src/pages/Game.js [hunk 2]
```diff
@@ -29,24 +66,185 @@ export default function Game() {
...
   }
 
+  const classes = useStyles();
+
   return (
     <div>
-      <Navbar/>
-            Game {gameId}
-      <AuthCheck>
+      <Navbar joinRoom={joinRoom} roomId={roomId} setRoomId={setRoomId}/>
```
Comment (danieltre23) [inline ubg/src/pages/Game.js:73]: Did you change Navbar? I don't see the file in files changed

Comment (shandyk18) [reply ubg/src/pages/Game.js:73]: I was moving the join room to the navbar but it looked wack so I deleted it. I missed deleting these though!

**Pull request merged**